drivers/power/adp1050/adp1050.c

```diff
@@ -5,30 +5,36 @@
 ********************************************************************************
  * Copyright 2024(c) Analog Devices, Inc.
  *
+ * All rights reserved.
```

Comment:
The current license for all No-OS drivers is BSD 3 clause. Please leave it as it currently is in the adpd1050.c file.